assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -22,6 +22,21 @@ angular.module("openshiftConsole")
 
     $scope.projectName = $routeParams.project;
     $scope.sourceURLPattern = SOURCE_URL_PATTERN;
+    $scope.imageName = $routeParams.imageName;
+
+    $scope.breadcrumbs = [
+      {
+        title: $scope.projectName,
```

Comment:
Why is this necessary since the route is already containing the name ? Just want to understand the point :)